internal/conf/configuration.go

```diff
@@ -503,12 +503,12 @@ func (e *ExtensibilityPointConfiguration) ValidateExtensibilityPoint() error {
 		return validatePostgresPath(u)
 	case "http":
 		hostname := u.Hostname()
-		if hostname == "localhost" || hostname == "127.0.0.1" || hostname == "::1" {
-			return validateHTTPSHookSecrets(e.HTTPHookSecrets)
+		if hostname == "localhost" || hostname == "127.0.0.1" || hostname == "::1" || hostname == "host.docker.internal" {
```

Comment:
To allow for local development, so that Auth can reach the Edge function which runs in a container